web-api/src/storage/postgres.rs

```diff
@@ -121,13 +151,15 @@ impl Database {
                 .push_values(
                     documents.by_ref().take(Self::BIND_LIMIT / 6),
                     |mut builder, document| {
+                        let embeddings: Vec<_> = document.embeddings.iter().map(Json).collect();
```

Comment:
:art: we have `itertools` at our disposal
```suggestion
                        let embeddings = document.embeddings.iter().map(Json).collect_vec();
```

Reply:
I still prefer the other style, but I also normally do not use inlay hints as I find them too verbose. I changed it.